### ISSUE TITLE
Scrollbar does not displayed in textarea or container with overflow auto in IE11 when it is added inside <perfect-scrollbar>

### ISSUE THREAD

**Issue by pavandixit93** [zefoy/ngx-perfect-scrollbar#246]
I have requirement of adding textarea in perfect-scrollbar, I tried adding textarea but the problem is that i am unable to see scrollbar in textarea, this is working fine in Chrome.
Please find example: https://ngx-perfect-scrollbar-khyofm.stackblitz.io. 
-- Add long text in textarea, you wont see any scrollbar like it shows in Chrome.

**Comment by sconix** [comment]
Hard to think of anything in this library that could cause that. Most likely weird IE11 bug that might be fixable by some specific styles.

**Comment by pavandixit93** [comment]
@sconix  Thanks for the quick response. 
Textarea without <perfect-scrollbar> in the html works as per expected.
but when added inside perfect-scrollbar does not work, Do you have any idea, due any CSS property inside `perfect-scrollbar` hiding scrollbar of textarea. 
_Again one thing, this is not only for textarea, In below example i have added small `div` with overflow scroll and height. If we see there is no scrollbar inside that div in IE11. and when i tested it in Chrome it shows the scrollbar._
_In short anything scrollable inside perfect-scrollbar in IE11 doesn't show the scrollbars._
Example: https://stackblitz.com/edit/ngx-perfect-scrollbar-khyofm?file=src%2Fapp%2Fapp.component.html

**Comment by sconix** [comment]
I have no idea what is causing it, at least the inspect of IE11 did not show that any styles would be "leaking" to the textarea that could cause it.

Sadly I have no interest looking into it further since nobody really should be using IE11 and we only fix critical IE11 related issues and that sort of bug would not fall into this category so even if this kind of problem exists in our product we would not be fixing it.

If you find the cause please share though, naturally if its an easy fix or PR is provided I will include it.

**Comment by pavandixit93** [comment]
Ok. I will give try.

**Comment by pavandixit93** [comment]
@sconix I just got the fix. The cause was due to `.ps` class used inside ngx-perfect-scrollbar.
when i checked its css, it has `-ms-overflow-style: 'none'`, this was causing to hide scrollbar in IE. So to fix it at our end i have added small fix like `.ps textarea { -ms-overflow-style: 'scrollbar' }`. This works perfectly.

**Comment by sconix** [comment]
Thanks! I look into this as soon as I have time if I can corporate the fix to the library itself and it wont cause any other issues.
